android/build.gradle

```diff
@@ -27,7 +27,7 @@ android {
     buildToolsVersion getExtOrDefault('buildToolsVersion')
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion getExtOrIntegerDefault('minSdkVersion')
```

Comment:
You can't imagine how hard this was to figure out haha

Reply:
😬